bin/olly.ml

```diff
@@ -225,6 +227,17 @@ let () =
     Arg.(value & flag & info [ "json" ] ~docv:"json" ~doc)
   in
 
+  let output_option =
+    let doc =
+      "Redirect the output of `olly` to specified file. The output of the \
```

Comment:
What does "output of the command is not redirected" mean?

Reply:
If `prog.exe` prints on stdout and/or stderr, then `olly latency -o some_file prog.exe` will print the latency info in `some_file`, but the original prints will still go to stdout/stderr

Reply:
It might be useful to add a description in the PR message to indicate how this command works like ^^.